designer/client/random-id.tsx

```diff
@@ -0,0 +1,12 @@
+import { customAlphabet } from "nanoid";
+
```

Comment:
this should just be named a .ts file - we should just reserve .tsx for those with tsx in.

Reply:
Good catch 👍